src/components/PeopleFilters.tsx

```diff
@@ -1,18 +1,44 @@
+import { useSearchParams } from "react-router-dom";
+import { SearchLink } from "./SearchLink";
+import classNames from "classnames";
+
 export const PeopleFilters = () => {
+  const [searchParams, setSearchParams] = useSearchParams();
+
+  const sex = searchParams.get('sex') || null;
+  const centuries = searchParams.getAll('centuries') || [];
+  const query = searchParams.get('query') || '';
+
+  const resetFilters = {
+    sex: null,
+    centuries: null,
+    query: null,
+  };
```

Comment:
The `resetFilters` object sets all filter parameters to `null`. However, setting URL parameters to `null` might not remove them from the URL. Consider using an empty string or removing the parameters entirely to ensure they are reset properly.